### ISSUE TITLE
Dual Head: Cursor jumps back after moving to other display

### ISSUE THREAD

**Issue by wilecoyote2015** [swaywm/sway#4971]
* Sway Version:
sway version 1.4-ffbf10d0 (Feb  3 2020, branch 'master')

Upload the debug log to a pastebin service such as [gist.github.com](https://gist.github.com), and link to it below.

* Debug Log:
https://gist.github.com/wilecoyote2015/3a184955e84ff1fc5e95a477d31894b9

You should try to reproduce the issue with the default configuration. If you cannot, please reproduce with a minimal configuration, upload the config to a pastebin service, and link to it below.

* Configuration File:
https://gist.github.com/wilecoyote2015/6cedcfc37f65cec50115624dbc99a8b7

Finally, explain the steps you took in plain English to reproduce the problem below.
1. Start sway with two displays
2. Move the cursor from Display showing workspace 1 to the center of the display showing workspace 2.

After the cursor has passed approx. 1/4 of workspace 2, it jumps back to workspace 1. When moving the cursor to workspace 2 again, the issue does not occur, but a similar pattern is observed when trying to move from workspace 2 to workspace 1, then.

I am using Arch Linux on a VirtualBox Vm with VboxSVGA graphics adapter without 3D acceleration.

**Comment by wilecoyote2015** [comment]
Sometimes, the issue does not appear after resizing and re-maximizing the VM Windows in the Windows host. Hence, I assume that this is a problem with VirtualBox or Windows, and therefore not a bug of Sway.

**Comment by RedSoxFan** [comment]
Possible duplicate of #4454
